integrations/azure/share-to-azure-functions/README.md

```diff
@@ -0,0 +1,260 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# Reacting to Events from Vendia Share using Azure Functions
+
+In this example, we'll use a cross-cloud [Universal Application](https://www.vendia.net/product) to demonstrate real-time, event-driven processing using Azure Functions.  Consider the case of a Supplier (on AWS), working through a Distributor (on AWS), who delivers goods to a Retailer (on Azure).  When the Supplier makes an adjustment (say by changing the anticipated fulfilment date of an existing purchase order), both the Distributor and the Retailer would like to be made aware (and take action) immediately.
```

Comment:
I went with `Supplier` when talking about the Supplier itself, but left "Supplier Node", "Supplier's Node", and "Supplier's AWS Node" as they were to avoid weird code block breaks in the sentence.  Let me know if this works from a reader's perspective.